code/modules/vehicle/janicart.dm

```diff
@@ -115,6 +115,9 @@
 		to_chat(user,"<span class='notice'>You upgrade [src] with [I].</span>")
 		update_icon(UPDATE_OVERLAYS)
 		return
+	if(istype(I, /obj/item/borg/upgrade/vtec) && floorbuffer)
+		floorbuffer = FALSE
+		vehicle_move_delay -= buffer_delay
```

Comment:
You'd prob want a return after this. Else the ..() gets called

Reply:
This is intended, since installation of VTEC is handled in the parent, though I should probably check if VTEC is already installed.

Reply:
Prob better to explicitly do a `return ..()` in that case with an added comment. Esp since it can cause you to put the upgrade into the trashbag if it's big enough to hold it.